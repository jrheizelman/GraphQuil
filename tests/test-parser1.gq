void main()
{
print(4);
}