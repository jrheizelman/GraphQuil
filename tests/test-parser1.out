Void main()
{
print(4);
}